elide-core/src/main/java/com/yahoo/elide/graphql/PersistentResourceFetcher.java

```diff
@@ -64,6 +93,17 @@ public Object get(DataFetchingEnvironment environment) {
     }
 
     /**
+     * Checks whether sort/filter/pagination params are passed w unsupported operation
+     * @param environment environment encapsulating graphQL's request environment
+     */
+    private void filterSortPaginateSanityCheck(Environment environment) {
+        if(environment.filters.isPresent() || environment.sort.isPresent() || environment.offset.isPresent()
+                || environment.first.isPresent()) {
+            throw new IllegalArgumentException("Pagination/Filtering/Sorting is only supported with FETCH operation");
```

Comment:
Down below we throw BadRequestException.  Here we throw IllegalArgumentException.  Let's be consistent.